tests/test_main.py

```diff
@@ -54,6 +59,8 @@ def test_load_and_validate_invalid(name: str, path: Path):
     # Either a DataPackSpecError or ValidationError is expected:
     if error_type == "DataPackSpecError":
         assert isinstance(exception_info.value, DataPackSpecError)
+    elif error_type == "DecodeError":
+        assert isinstance(exception_info.value, DecodeError)
```

Comment:
Do you think this is an improvement with regard to readability?

Reply:
In normal code a fair point. However, test code should be DAMP instead of DRY. While the DRYness is improved, I think the code becomes less DAMP. Anyhow both solutions would be OK thus I will include your suggestion.